client/src/app/primitives/modal/Modal.less

```diff
@@ -31,4 +31,17 @@
   border-radius: 2px;
   padding: 20px;
   z-index: 1001;
-}
\ No newline at end of file
+
+  .close {
+    vertical-align: middle;
+    font-size: 2em;
+    line-height: 2em;
```

Comment:
Effect of this is that the hover event fires well beyond the x button. Is this intended?

Reply:
Unintended positive side-effect I'd say. Will change once we move to a css framework foundation anyway. I'd not change that here.